test/dlc_tests/sanity/test_ecr_scan.py

```diff
@@ -272,6 +276,24 @@ def helper_function_for_leftover_vulnerabilities_from_enhanced_scanning(
         LOGGER.info(
             f"[NonPatchableVulns] [image_uri:{ecr_enhanced_repo_uri}] {json.dumps(non_patchable_vulnerabilities.vulnerability_list, cls= test_utils.EnhancedJSONEncoder)}"
         )
+    LOGGER.info(
```

Comment:
Flag `is_generic_image` is used by other non-DLC images to run the scans on their images. We do not want to upload the s3 data for those images. So please make sure that we are uploading the data only when `is_generic_image` is `False`.